README.md

```diff
@@ -229,9 +233,13 @@ You need to have Helm installed locally
 To deploy to a cluster you can run the following command
 
 ```
-helm tiller run helm upgrade --install survey-runner ./helm/runner --set projectId=PROJECT_ID --set submissionBucket=BUCKET_NAME 
+./scripts/deploy/runner.sh <PROJECT_ID> <SUBMISSION_BUCKET_NAME> <IMAGE_TAG>
 ```
-
+##### Example
+ ```
+ ./scripts/deploy/runner.sh census-eq-dev-1234567 census-eq-dev-1234567-survey-runner-submission v3.0.0
+ ```
+ 
```

Comment:
There is no mention of deploying credentials before deploying the app.

Reply:
Added.